lms/djangoapps/learner_dashboard/urls.py

```diff
@@ -1,9 +1,10 @@
 """Learner dashboard URL routing configuration"""
 from django.conf.urls import url
 
-from . import views
+from lms.djangoapps.learner_dashboard import programs, views
 
 urlpatterns = [
     url(r'^programs/$', views.program_listing, name='program_listing_view'),
+    url(r'^programs/fragment/$', programs.ProgramsFragmentView.as_view(), name='program_listing_fragment'),
     url(r'^programs/(?P<program_uuid>[0-9a-f-]+)/$', views.program_details, name='program_details_view'),
```

Comment:
Nit: same as above, since we may add more fragments down the road, we want to be descriptive about exactly which fragment we are referencing in the URL